### PR TITLE
Big ui revision.

### PR DESCRIPTION
**Pull request by gemini6**
Hey StevesBrain. If you read the IMPORTANT_NOTE.txt file it explains why I did this but feel free to use this or not. Its best viewed so far on small devices such as phones and tablets (under 768px) I made it semi-responsive so far. I love the header and footer on a phone or tablet but on larger screens I don't know what I want to do yet. It just doesn't look as good anymore to me.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
+hey tia
```
Comment (gemini6) [inline index.html]: I was showing my gf how commits work on GitHub. :P

**Comment by stevesbrain** [comment]
Thanks @gemini6 - will check these changes out in the next day or two; just on holidays at the moment! Thanks  in advance! :)

**Comment by gemini6** [comment]
No worries. Enjoy.

**Comment by gemini6** [comment]
@gemini6

Note: The sight is fully responsive now and viewable on any screen size. Cheers.

**Review by gemini6 — COMMENTED** [review]
The local copy of animate.css is not required as the cdn is available and being used. I just threw it in in case anyone wanted to use it locally instead.

**Comment by stevesbrain** [comment]
All merged - thanks @gemini6 :)

**Pull request merged**